spec/lib/healthcheck/government_data_check_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+RSpec.describe Healthcheck::GovernmentDataCheck do
+  describe "#status" do
+    context "government data is unavailable" do
+      it "returns critical" do
+        gov_data_check = Healthcheck::GovernmentDataCheck.new
+        expect(gov_data_check.status).to be(:critical)
+      end
+    end
+
+    context "government data is set to expire" do
+      it "gives a warning" do
+        BulkData::Cache.write("government-v1", "value")
```

Comment:
The use of this id makes an unnecessary coupling between this test and the constant in GovernmentRepository. I'd suggest instead using the helper instead: https://github.com/alphagov/content-publisher/blob/c074cd3ebb6c885f1fa3d7112cd7af6bbb4d01da/spec/support/bulk_data_helper.rb#L4

Reply:
Nice one, forgot that you could do this, have adjusted.